src/types/character/character-type.ts

```diff
@@ -0,0 +1,17 @@
+import { IStat } from './stat-type'
```

Comment:
The file name is suffixed by -type.
I do not think that this is good since the exported member should match the file resulting to add back the Type suffix on the interface name which we already defined that this is not good as well.
On the other hand if you like identify your exported members you could simply put this one into a folder called either interfaces or called models or called types.

Reply:
If you look at the whole file path, there already is a 'types/' folder as a parent.

I will remove the -type from the file names.